components/collection/unlockable/UnlockableLandingTag.vue

```diff
@@ -35,25 +36,32 @@
         </defs>
       </svg>
 
-      <span> {{ $t('mint.unlockable.mintLive') }} </span>
-      <NeoIcon
-        icon="horizontal-rule"
-        pack="fa-sharp"
-        size="small"
-        class="separator mx-2" />
+      <span> {{ mintLiveText }} </span>
     </div>
-
+    <div class="separator mx-2" />
     <nuxt-link
-      class="is-flex is-align-items-center has-text-weight-bold"
+      class="is-flex is-align-items-center has-text-weight-bold my-2"
       to="/stmn/drops/free-drop">
       {{ $t('mint.unlockable.takeMe') }}
     </nuxt-link>
   </div>
 </template>
 
 <script lang="ts" setup>
-import { NeoIcon } from '@kodadot1/brick'
 const isUnlockableLandingTagVisible = true
+const { $i18n } = useNuxtApp()
+
+const { width } = useWindowSize()
+
+const isSmallSize = computed(() => width.value < 502)
```

Comment:
very specific number... 502
plus I find the name confusing
maybe something like ` smallWidth` or something?

Reply:
Number `502` is from the description of the original issue.

As for the naming, it's ok to change.

Reply:
Yup, that's where the pill shape announcement changes its layout to the two-row